src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveRenderedService.cs

```diff
@@ -130,9 +147,20 @@ private static AcceptHeader GetValidRenderAcceptHeader(IReadOnlyCollection<Accep
         {
             throw new NotAcceptableException(DicomCoreResource.MultipleAcceptHeadersNotSupported);
         }
-        else if (acceptHeaders.Count == 1 && acceptHeaders.First().MediaType != null && !StringSegment.Equals(acceptHeaders.First().MediaType, KnownContentTypes.ImageJpeg, StringComparison.InvariantCultureIgnoreCase))
+
+        if (acceptHeaders.Count == 1)
         {
-            throw new NotAcceptableException(DicomCoreResource.NotAcceptableHeaders);
+            var mediaType = acceptHeaders.First()?.MediaType;
+
+            if (mediaType == null || (!StringSegment.Equals(mediaType.ToString(), KnownContentTypes.ImageJpeg, StringComparison.InvariantCultureIgnoreCase) && !StringSegment.Equals(mediaType.ToString(), KnownContentTypes.ImagePng, StringComparison.InvariantCultureIgnoreCase)))
+            {
+                throw new NotAcceptableException(DicomCoreResource.NotAcceptableHeaders);
+            }
+
+            if (StringSegment.Equals(mediaType.ToString(), KnownContentTypes.ImagePng, StringComparison.InvariantCultureIgnoreCase))
+            {
+                return new AcceptHeader(KnownContentTypes.ImagePng, PayloadTypes.SinglePart);
+            }
         }
 
         return new AcceptHeader(KnownContentTypes.ImageJpeg, PayloadTypes.SinglePart);
```

Comment:
I thought accept header is mandatory.

Reply:
No, per Matt if they dont specify we return a jpeg